ihmc-perception/src/main/java/us/ihmc/perception/heightMap/HeightMapMessageTools.java

```diff
@@ -103,16 +93,18 @@ public static void toMessage(HeightMapData heightMapData, HeightMapMessage messa
       for (int i = 0; i < heightMapData.getNumberOfOccupiedCells() && i < messageToPack.getKeys().capacity(); i++)
       {
          int key = heightMapData.getKey(i);
-         messageToPack.getKeys().add(key);
-         messageToPack.getHeights().add((float) heightMapData.getHeightAt(key));
+         messageToPack.getKeys().add((short) key);
```

Comment:
Why the cast to short?

```suggestion
         messageToPack.getKeys().add(key);
```